apps/evm/src/clients/api/queries/useGetPools/getPools/getApiPools/index.ts

```diff
@@ -10,7 +10,11 @@ export interface ApiRewardDistributor {
   supplySpeed: string;
   borrowSpeed: string;
   priceMantissa: string;
-  rewardsDistributorContractAddress: string | null;
+  rewardsDistributorContractAddress: string;
+  rewardType: 'venus_isolated_market' | 'venus_legacy_market' | 'merkl';
```

Comment:
Go be consistent with the rest of the codebase, these should be written in camel case.